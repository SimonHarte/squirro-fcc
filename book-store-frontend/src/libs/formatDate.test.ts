import formatDate from './formatDate';

test('should format ISO date to common readable', () => {
  const isoDate = new Date(1990, 4, 1).toISOString();

  expect(formatDate(isoDate)).toMatchSnapshot();
});
