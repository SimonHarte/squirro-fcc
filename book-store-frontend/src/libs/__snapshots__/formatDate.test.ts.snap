// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should format ISO date to common readable 1`] = `"01.05.1990"`;
